technical-docs/designs/repos-merge.md

```diff
@@ -0,0 +1,285 @@
+    ├── ssh-url
+├── .editorconfig
+├── .env.example
+├── .gitignore
+├── CHANGELOG.md
+├── README.md
+├── azure-pipelines.yml
+├── jest.config.js
+├── package.json
+├── release-pipeline.yml
+├── smoke-test-pipeline.yml
+├── spk-config.yaml
+├── tsconfig.json
+├── tslint.json
+├── typings
+├── webpack.config.js
+├── .yarn.lock
+```
+
+#### Themes
+
+The main themes are related to organizing `docs` and `build` related artifacts
+in `spk` from the root to appropriate directories.
+
+1. Oragnize all pipeline yaml files, scripts, and tools that are related to
+   building and releasing binaries under `build` directory.
+
+2. Organize all docs that are related to using and contributing bedrock under
+   `docs` directory.
+
+#### Proposed Changes
+
+The majority of the proposed changes in `spk` repo are related to organizing
+`docs` and `build` folders.
+
+- Move following files from the `root` to `build\pipelines` folder and make sure
+  all pipelines continue to work.
+  - azure-pipelines.yml
+  - release-pipeline.yml
+  - smoke-test-pipeline.yml
+- Move following files from the `scripts` to `build\tools` folder and verify the
+  scripts generate `command` docs.
+  - generateDoc.ts
+  - locateAliases.ts
+- Move `patches` directory to `build\patches` folder and make sure `yarn build`
+  and `yarn test` continue to work.
+- Move `azure-pipelines\templates` folder to `build\pipelines\templates` folder
+  and verify all pipelines continue to work.
+- Move `guides\contributing.md` file to `docs\contribution` folder
+- Move `guides` folder to `docs` folder
+- Move `technical-docs\designs` to `docs\contribution\designs` folder
+- Remove `.github\workflows` folder since it is not being used
+
+#### Testing Changes
+
+After making the above changes, need to make sure that the follwing tests pass.
+
+- Verify cli build pipeline is working
+- Verify cli integration test pipeline is working
+- Verify cli release pipeline is working
+- Verify cli autogenerated docs are working on GitHub pages
+- Verify all docs links are working
+
+#### SPK repo after changes
+
+`spk` repo would look like below after implementing the above changes.
+
+```
+├── build
+    ├── pipelines
+        ├── templates
+        ├── azure-pipelines.yml
+        ├── release-pipeline.yml
+        ├── smoke-test-pipeline.yml
+     ├── patches
+        ├── 001-azure-devops-node.patch
+     ├── tools
+        ├── generateDoc.ts
+        ├── locateAliases.ts
+        ├── release-version-bump.sh
+        ├── tag-release.sh
+        ├── update_introspection.sh
+├── docs
+    ├── commands
+    ├── contribution
+        ├── contributing.md
+        ├── designs
+    ├── guides
+├── src
+    ├── lib
+    ├── commands
+    ├── logger
+├── tests
+├── typings
+    ├── ssh-url
+├── .editorconfig
+├── .gitignore
+├── jest.config.js
+├── tsconfig.json
+├── tslint.json
+├── CHANGELOG.md
+├── README.md
+├── package.json
+├── webpack.config.js
+├── .yarn.lock
+```
+
+## Phase 2: Move SPK to Bedrock Repo
```

Comment:
@sarath-p Let's add an (under review) qualifier to Phase 2 and beyond. I think we can agree on Phase 1 and understand that Phase 2 and beyond are still being thought about.